src/app.js

```diff
@@ -156,15 +159,39 @@ var run = function () {
   chromeCloudSync()
 
   // ----------------- editor ----------------------
-
   var editor = new Editor()
-
-  // ----------------- tabbed menu -------------------
+  // -------------- fileExplorer ------------------
+  // @NOTE HTML#ID creates window['<ID name>'] element as a global
+  // var api = fileExplorer(window.treeview, files, editor)
+  fileExplorer(window.treeview, files, editor)
```

Comment:
I'm fine with using factory methods, but it should be visible from their name that they are factory methods.